books/RayTracingTheNextWeek.html

```diff
@@ -656,40 +660,67 @@
 
 </div>
 
-
-An Optimized AABB Hit Method
------------------------------
-In reviewing this intersection method, Andrew Kensler at Pixar tried some experiments and proposed
-the following version of the code. It works extremely well on many compilers, and I have adopted it
-as my go-to method:
+<div class='together'>
+The new code above introduces new interval functions we need to write: `interval::is_empty()`,
+`interval::intersect()`, and `span()`.
```

Comment:
`interval::span()` to match the others?

Reply:
`span()` is a standalone function.

Reply:
You're right, I missed that. I updated my suggestion above to take that into account.